t/24-worker-jobs.t

```diff
@@ -1369,8 +1369,6 @@ subtest 'asset upload' => sub {
 };
 
 subtest 'log file upload' => sub {
-    my $utils_mock = Test::MockModule->new('OpenQA::Utils');
-    $utils_mock->noop('wait_with_progress');
```

Comment:
Hm, this likely means unnecessary sleeping in tests unless you mock the sleep call

Reply:
The test runs about 4 to 5 seconds slower.

Reply:
Since i don't care much for an argument about mocking or not, please just tell me how you want the sleep to be mocked.

Reply:
To make this easy I've reused the same ugly global hack we already use in another test. https://github.com/os-autoinst/openQA/pull/4062/commits/bfd335968bd4fb1edb32263acf39637383334878